docs/ref/databases.txt

```diff
@@ -719,7 +719,7 @@ you add quotes where necessary before copying a query into an SQLite shell.
 Oracle notes
 ============
 
-Django supports `Oracle Database Server`_ versions 11.1 and higher. Version
+Django supports `Oracle Database Server`_ versions 11.2 and higher. Version
```

Comment:
Have we done something to break support for Oracle 11.1? We documented 11.1 as the minimum version in Django 1.8 -- if we want to drop support for 11.1 we also need to document that in the release notes.